assets/js/hooks/useScreenplayContext.tsx

```diff
@@ -255,9 +255,15 @@ const ScreenplayProvider = ({ children }: Props) => {
 export {
   useScreenplayContext,
   useScreenplayDispatchContext,
-  usePlacesPageContext,
-  usePlacesPageDispatchContext,
-  useAlertsPageContext,
-  useAlertsPageDispatchContext,
+  usePlacesListContext,
+  usePlacesListDispatchContext,
+  useAlertsListContext,
+  useAlertsListDispatchContext,
   ScreenplayProvider,
+  FilterValue,
```

Comment:
I know all of these exports are necessary, but since you mentioned not liking how many there are, maybe it would help to split it into two `export` statements—one for values, one for types?

```tsx
// types
export {
  FilterValue,
  DirectionID,
  PlacesListReducerAction,
  ...
};

// values
export {
  usePlacesListContext,
  usePlacesListDispatchContext,
  ...
};
```

Imports in other modules could similarly be split into two lines, or not.

Reply:
Love this idea.